nanome/_internal/_process/_process_manager.py

```diff
@@ -84,18 +85,18 @@ def enqueue_output(pipe, queue, text, bufsize):
             # Log process settings
             exec_path = request.executable_path
             session_id = entry.session._session_id
+            id = request.label if request.label else exec_path
```

Comment:
```suggestion
            id = request.label or exec_path
```
`a if a else b` can be simplified to `a or b`